plugins/meta/sbr/main.go

```diff
@@ -253,12 +253,23 @@ func doRoutes(ipCfgs []*current.IPConfig, iface string) error {
 		}
 
 		log.Printf("Source to use %s", src.String())
-		rule.Src = &src
+		srcRule.Src = &src
 
-		if err = netlink.RuleAdd(rule); err != nil {
-			return fmt.Errorf("Failed to add rule: %v", err)
+		if err = netlink.RuleAdd(srcRule); err != nil {
+			return fmt.Errorf("Failed to add src rule: %v", err)
 		}
 
+		// Only add an interface rule if there is 1 IP address configured on the interface
+		if len(ipCfgs) == 1 {
```

Comment:
This rule doesn't reference `ipCfg` at all, so why is this in the `for _, ipCfg := range ipCfgs {` loop?

Reply:
That is a very good question :) I can move it out of the loop.